repo/packages/Z/zeppelin/1/marathon.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
since zeppelin will be launched on a private slave with this app definition, how will users access it?  Don't we need marathon-lb support?